src/main/resources/conf/global-default.xml

```diff
@@ -0,0 +1,200 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="configuration.xsl"?>
+
+<configuration>
+
+  <!-- Application configurations -->
+  <property>
+    <description>Default queue to submit to YARN.</description>
+    <name>shifu.yarn.queue</name>
+    <value>default</value>
+  </property>
+
+  <property>
+    <description>Name of your YARN application.</description>
+    <name>shifu.application.name</name>
+    <value>TensorFlowApplication</value>
+  </property>
+
+  <property>
+    <description>Max runtime of the application before killing it, in milliseconds.</description>
+    <name>shifu.application.timeout</name>
+    <value>0</value>
+  </property>
+
+  <property>
+    <description>The path of app master in local</description>
+    <name>shifu.application.app-jar-path</name>
+    <value>/x/home/webai/tensorflow/shifu-tensorflow-on-yarn-0.0.1-SNAPSHOT.jar</value>
+  </property>
+
+  <property>
+    <description>python env path in hdfs</description>
+    <name>shifu.application.python-env-zip</name>
+    <value>/user/website/.tensorflow_yarn/python_env.zip</value>
+  </property>
+
+  <property>
+    <description>python env path</description>
+    <name>shifu.application.python-binary-path</name>
+    <value>/x/home/website/python2.7</value>
+  </property>
```

Comment:
local python, do we need it now?

Reply:
right now python and glibc are in hdfs.
this is just path inside of the zip.